### PR TITLE
[update] add application and env names to the logging

### PR DESCRIPTION
**Pull request by denysov-serhii**

### CODE REVIEW

File: Application.php [hunk 1]
```diff
@@ -169,15 +181,28 @@ private function handleException(Request $request, int $type, \Exception $except
...
             throw new HttpException(Response::HTTP_INTERNAL_SERVER_ERROR, 'Internal server error');
         }
 
         throw $exception;
     }
+
+    /**
+     * @return ApplicationLogService
+     */
+    private function getApplicationLogService(): ApplicationLogService
```
Comment (andremit) [inline Application.php:201]: Please move to container

File: Services/ApplicationLogService.php [hunk 2]
```diff
@@ -7,14 +7,29 @@
...
     /**
      * @return array
      */
     protected function loggersMap(): array
     {
         return [
-            self::LOG_TO_FILE => function () {return (new FileLoggingRegistry())->logger(LOG_PATH . 'application.log', 'application');},
-            self::LOG_TO_CLOUD_WATCH => function () {return (new CloudWatchLoggingRegistry())->logger('php-logs', 'application', 'core');},
+            self::LOG_TO_FILE => function () {return (new FileLoggingRegistry())->logger(LOG_PATH . $this->streamName . '.log', 'application');},
+            self::LOG_TO_CLOUD_WATCH => function () {return (new CloudWatchLoggingRegistry())->logger('php-logs', $this->streamName, 'core');},
```
Comment (andremit) [inline Services/ApplicationLogService.php:32]: can you do this configurable ? 
this is hardcode

**Review by andremit — APPROVED** [review]

File: Application.php (hunk 1 above)
Comment (denysov-serhii) [reply Application.php:201]: moved

File: Services/ApplicationLogService.php (hunk 2 above)
Comment (denysov-serhii) [reply Services/ApplicationLogService.php:32]: it  makes additional unjustified complexity at this stage, we can do it when to appear a necessity

**Pull request merged**